Modules/Scripted/DICOM/Resources/UI/DICOM.ui

```diff
@@ -6,7 +6,7 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>372</width>
+    <width>615</width>
```

Comment:
Keep this value the same. Larger value may increase minimum width of the DICOM module.
```suggestion
    <width>372</width>
```